atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleDdlTable.java

```diff
@@ -259,7 +260,11 @@ public void compactInternally() {
     }
 
     private String getPrimaryKeyConstraintName(String tableName) {
```

Comment:
getPrimaryKeyConstraintName and truncateToMaxOracleLength are copied through several classes in this PR.  Can we have just a single implementation?

Could move to a utilities class or pull up to shared common parent class or interface.